### ISSUE TITLE
Clarify type of ``vcf_zarr_version`` attr

### ISSUE THREAD

**Issue by jeromekelleher** [sgkit-dev/vcf-zarr-spec#16]
I'm assuming it's a string, but it could be interpreted as a float

**Comment by tomwhite** [comment]
Yes, it's a string. In general versions are not floats (e.g. 0.2.1).

**Comment by jeromekelleher** [comment]
I think we just need to put some quotes around the value to make that more obvious.
